rule/var_naming.go

```diff
@@ -22,12 +22,26 @@ var knownNameExceptions = map[string]bool{
 	"kWh":          true,
 }
 
+// extraBadPackageNames is the list of "bad" package names from https://go.dev/wiki/CodeReviewComments#package-names
+// and https://go.dev/blog/package-names#bad-package-names.
+// The rule warns about the usage of any package name in this list if skipPackageNameChecks is false.
+// Values in the list should be lowercased.
+var extraBadPackageNames = map[string]struct{}{
```

Comment:
I wouldn't have renamed this one with the prefix extra, they are the default ones

The main reason is that it creates a confusion when reading `r.extraPackageNames` and `extraPackagesNames`

So here I would suggest this

```suggestion
var badPackageNames = map[string]struct{}{
```


```suggestion
var defaultBadPackageNames = map[string]struct{}{
```

Everything else is fine